src/index.ts

```diff
@@ -160,6 +161,20 @@ function main() {
                 console.log('Please login with an account that has at least one active subscription');
             } else {
                 const questions: IQuestions = new Questions(program.environment);
+                // If user didn't specify any release version then prompt from the version
+                // that have been released so far. Also note that this list will need to be
+                // updated with each release. More long term solution would be to look at github
+                // tags/releases to populate the list of options shown to the user per solution
+                // type.
+                if (!program.release) {
+                    questions.addQuestion({
+                        choices: [{value: '1.0.0-preview'}, {name: 'latest', value: 'testing'}],
```

Comment:
can we use a different name for the development build? "latest" sounds like a stable release, I would suggest "last development build" or similar

Reply:
i could just call it 'development'